NEWS.md

```diff
@@ -1,8 +1,14 @@
 # simstudy (development version)
 
 ## New features
-* Added double-dot (dynamic) functionality to defSurv. Users can now specificy 
+* Added double-dot (dynamic) functionality to defSurv. Users can now specify 
 double-dot variables in scale, shape, and formula parameters.
+* It is possible to generate variable cluster sizes using the *clusterSize*
+distribution in `defData` and `defDataAdd`.
+
+## Major fix
+* Updated the random effect variance generation for function `iccRE` under the
+Poisson distribution.
```

Comment:
What was the issue here? Was there a bug that might have influenced existing code/results?

Reply:
Looking at the code below "improved" might be clearer  than updated?

Reply:
Yeah - I was trying to find the source of my code - it looked really odd (though I know I did it). I found the source that I thought I was using (it is now referenced in the iccRE.md file), and it was quite different and simpler than what I was doing. It also has the benefit of being correct. Though the ICC is not such an obvious concept in the context of the Poisson distribution, so I am not sure if the one I generated was wrong or just a different definition. This isn't great, but I just can't figure out where the old method came from.